src/reuse/modules/nonUi5/element.ts

```diff
@@ -423,6 +423,20 @@ export class ElementModule {
     return this.isPresentByCss(xpath, index, timeout);
   }
 
+  /**
+   * @function isSelected
+   * @memberOf nonUi5.element
+   * @description Returns a boolean if the element (e.g. checkbox) is selected.
+   * @param {Object} elem - The element.
+   * @returns {boolean}
+   * @example const elem = await nonUi5.element.getById("elem01");
+   * const isSelected = await nonUi5.element.isSelected(elem);
+   */
+  async isSelected(elem: Element): Promise<boolean> {
+    const vl = this.vlf.initLog(this.isSelected);
+    return elem.isSelected();
```

Comment:
Why are we wrapping wdio element method, for better visibility to teams?

Reply:
- we wrap everything, to be able to keep compatibility (same as moving from vyper to qmate, or if wdio does changes)
- to keep consistent usage (we have function where we pass the elem as argument, wdio has functions on elements)
- to provide flexibility and additional checks (in this case only the vl)
- transparency through Reuse API